www/components/core/Search.tsx

```diff
@@ -36,7 +37,14 @@ const SearchResultItem = styled.div`
   }
 `;
 
-const ItemLink = styled(RouterLink)`
+const ItemRouterLink = styled(RouterLink)`
+  display: block;
+  padding: ${(props) => props.theme.spacing.xs};
+
+  width: 100%;
+`;
+
+const ItemLink = styled(Link)`
```

Comment:
A nicer way to do this would be:

```
const linkStyles = css`
  display: block;

  width: 100%;
  padding: ${(props) => props.theme.spacing.xs};
`; 

const ItemLink = styled(RouterLink)`
  ${linkStyles}
`;
const ItemRouterLink = styled(RouterLink)`
  ${linkStyles}
`
```